_posts/2024-06-10-dgtal-release1.4.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: DGtal release 1.4
+---
+
+We are happy to announce the [release  1.4 of DGtal](https://github.com/DGtal-team/DGtal/releases/tag/1.4) and its tools. As always,many new features, edits and bugfixes are listed in the [Changelog](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md), and we would like to thank all devs involved in this release. In this short review, we would like to only focus on selected new features.
+
+* **New Features**:
+  * 
```

Comment:
* implicit shapes can be constructed from points clouds using winding numbers
* meshes can be updated through flip operations
* lattice polytopes describing segments or triangles can be constructed in a much more efficient manner
* DGtal requires C++17 now